src/Simplex/Chat/Operators.hs

```diff
@@ -306,6 +340,16 @@ newUserServer_ :: Bool -> Bool -> ProtoServerWithAuth p -> NewUserServer p
 newUserServer_ preset enabled server =
   UserServer {serverId = DBNewEntity, server, preset, tested = Nothing, enabled, deleted = False}
 
+presetSuperpeer :: Bool -> Text -> [Text] -> ConnReqContact -> NewUserSuperpeer
+presetSuperpeer = newSuperpeer_ True
+
+newSuperpeer :: Text -> [Text] -> ConnReqContact -> NewUserSuperpeer
+newSuperpeer = newSuperpeer_ False True
+
+newSuperpeer_ :: Bool -> Bool -> Text -> [Text] -> ConnReqContact -> NewUserSuperpeer
+newSuperpeer_ preset enabled name domains address =
```

Comment:
```suggestion
newSuperpeer_ preset enabled name domains !address =
```

Reply:
the point is to make it crash on invalid address - check that it does.

Reply:
it did produce error on invalid addresses even without it, it's created as `either error id $ strDecode ...`. but adding it